typed/clj.checker/src/typed/clj/checker/parse_unparse.clj

```diff
@@ -728,7 +728,7 @@
 (declare parse-function)
 
 (defn parse-fn-intersection-type [[Fn & types] opts]
-  (r/FnIntersection-maker (into [] (mapcat #(parse-function % opts)) types)))
+  (r/FnIntersection-maker (into [] (comp (map #(parse-function % opts)) cat) types)))
```

Comment:
What's wrong with `mapcat`?

Reply:
My bad, sorry. I missed that it was already a transducer version.